pkg/util/common.go

```diff
@@ -977,13 +977,16 @@ func IsClusterRoleRuleExist(oldRules []rbacv1.PolicyRule, newRule rbacv1.PolicyR
 
 // GetRouteClient attempts to get a Route Client. It returns nil if it
 // fails due to an error or due to being on kubernetes (doesn't support routes)
-func GetRouteClient(restConfig *rest.Config) *routeclient.RouteV1Client {
+func GetRouteClient(restConfig *rest.Config) (*routeclient.RouteV1Client, error) {
```

Comment:
remove error from the return statement

Reply:
GetRouteClient logs errors instead of returning them